### DISCUSSION TITLE
When is Behave 1.3.0 (aka: 1.2.7) GA?

### DISCUSSION THREAD

**Discussion by heysarthak** [behave/behave#1204] [Q&A] [answered]
Hi Team,

I know you are working on 1.2.7 dev pre-releases, When can we expect stable release for 1.2.7? 

Regards
Sarthak

**Comment by jenisys** [comment, marked as answer]
The current `behave v1.2.7.devX` pre-releases will lead to `behave v1.3.0`.
`behave v1.3.0` will be released before the end of the year (or earlier).

NOTES:
* The `behave v1.2.7.devX` are actually rather stable.
* If you are a normal behave-user and not rely on any `behave` internals, I would suggest that you try to use the newest pre-release.
* If stumble over something, let me know.

**Comment by CharlesPerrotMinot** [comment]
Hi! So... 1.3.0? Soon ish?

**Comment by jenisys** [comment]
SEE: https://pypi.org/project/behave/1.3.0/
